plugins/statsd/statsd_input_test.go

```diff
@@ -166,7 +166,7 @@ func TestParseMessage(t *testing.T) {
 	}
 
 	for msg, expected := range testData {
-		obtained, err := parseMessage([]byte(msg + "\n"))
+		obtained, _, err := parseMessage([]byte(msg + "\n"))
```

Comment:
Probably a good idea to capture this value here (and below) and check that it's what we expect.